akka-remote-tests/src/multi-jvm/scala/akka/remote/routing/RemoteScatterGatherSpec.scala

```diff
@@ -81,6 +81,9 @@ class RemoteScatterGatherSpec(multiNodeConfig: RemoteScatterGatherConfig) extend
         val connectionCount = 3
         val iterationCount = 10
 
+        // let them start
+        Thread.sleep(2000)
```

Comment:
The test was not failing but it had problems terminating the actor system, and that was because there was a race between starting the routees and immediately stopping them (the actor systems). All messages were typically served by one routee, and that is enough for scatter gather.